src/django/api/serializers.py

```diff
@@ -643,6 +644,61 @@ def get_contributor_fields(self, facility):
 
         return assign_contributor_field_values(list_item, fields)
 
+    def get_extended_fields(self, facility):
```

Comment:
Just curious - why is the `get_extended_fields` moved from `FacilityDetailsSerializer(FacilitySerializer)` to be under the `FacilitySerializer(GeoFeatureModelSerializer)`